components/input/input.css

```diff
@@ -0,0 +1,50 @@
+.input-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    background-color: var(--grey);
+    padding: 1rem;
+    border-radius: 4px;
+    margin-top: 1rem;
+}
+
+.fname{
+    width: 100%;
+    padding: 0.5rem 1rem;
+    margin: 0.5rem 0;
+    border: 1px solid #ccc;
```

Comment:
You can use variable for color here as well.

Reply:
i will change it now, thankyou for pointing it out Captain :)